src/modules/comment.js

```diff
@@ -1,18 +1,19 @@
-// eslint-disable-next-line import/no-cycle
+/* eslint-disable  import/no-cycle */
+
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.

Reply:
Bro have only disabled  one which was bringing an error,  Your Consideration will be highly appreciated as I have come far doing this project.

Reply:
I can show you on zoom how the code is working.

Reply:
Hello @zairdon20, let's see how we can solve the issue. Checkout this [article](https://dev.to/dan_v/dependency-cycle-detected-in-react-component-index-file-8o8), I think it will help to identify the cause to the error. Let me know if you need more help.

Reply:
Okay